src/util/linkify.js

```diff
@@ -12,8 +12,9 @@ const createLink = options => href => {
 	const target = options.target || '';
 	const targetAttr = `target="${target}"`;
 	const relAttr = target === '_blank' ? 'rel="noopener noreferrer"' : '';
+	const link = (href.search('www.') === 0) ? `http://${href}` : href;
```

Comment:
probably better to check for the _absence_ of the protocol rather than the _presence_ of `www.`. I guess the URL matcher won't match on other subdomain strings, but this is an indirect test.